mteb/tasks/Image/ImageTextPairClassification/AROVisualAttribution.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from mteb.abstasks.Image.AbsTaskImageTextPairClassification import (
+    AbsTaskImageTextPairClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class AROVisualAttribution(AbsTaskImageTextPairClassification):
+    images_column_names = ["image"]
+    texts_column_names = ["true_caption", "false_caption"]
+
+    metadata = TaskMetadata(
+        name="AROVisualAttribution",
+        description="Compositionality Evaluation of images to their captions.",
```

Comment:
Might be worth expanding this a little bit - had to look at the paper to understand it